component-library/components/login-manager.mdx

```diff
@@ -0,0 +1,281 @@
+
+## Usage
+
+The following code snippet is a minimal example of how to use the LoginManager component. Here, we pass in functions that will be invoked when the login process is complete (`onLoginCompleted`), or when a user clicks "Sign Up" (`onRedirectToSignup`) or "Forgot Password" (`onRedirectToSignup`).
+
+```jsx
+import * as React from "react";
+import { useNavigate } from "react-router-dom";
+import { LoginManager } from "@propelauth/react";
+
+export default function LoginPage() {
+  const navigate = useNavigate();
+
+  return (
+    <div>
+      <LoginManager
+        onLoginCompleted={() => navigate("/dashboard")}
+        onRedirectToSignup={() => navigate("/signup")}
+        onRedirectToForgotPassword={() => navigate("/forgot-password")}
+      />
+    </div>
+  );
+}
+```
+
+Now let's style the inner components of the LoginManager—we can do this by passing in an appearance object for each inner component as a prop. The appearance object lets you apply styles and classes to elements in a component, or even completely override those elements.
+
+```jsx
+import * as React from "react";
+import { useNavigate } from "react-router-dom";
+import { LoginManager } from "@propelauth/react";
+import { CustomCodeInput } from "./custom-components";
+
+export default function LoginPage() {
+  const navigate = useNavigate();
+
+  const loginAppearance = {
+    options: {
+      divider: false,
+    },
+    elements: {
+      Container: { backgroundColor: "#eee" },
+      SubmitButton: "btn-blue",
+    },
+  };
+
+  const verifyMfaAppearance = {
+    options: {
+      submitButtonText: "Verify",
+    },
+    elements: {
+      Container: { backgroundColor: "#eee" },
+      CodeInput: CustomCodeInput,
+      SubmitButton: "btn-yellow",
+    },
+  };
+
+  const completeAccountAppearance = {
+    elements: {
+      UsernameLabel: "custom-label",
+      UsernameInput: "custom-input",
+      SubmitButton: "btn-green",
+    },
+  };
+
+  return (
+    <div>
+      <LoginManager
+        onLoginCompleted={() => navigate("/dashboard")}
+        onRedirectToSignup={() => navigate("/signup")}
+        onRedirectToForgotPassword={() => navigate("/forgot-password")}
+        loginAppearance={loginAppearance}
+        verifyMfaAppearance={verifyMfaAppearance}
+        completeAccountAppearance={completeAccountAppearance}
+      />
+    </div>
+  );
+}
+```
+
+## Props
+
+<ParamField path="onLoginCompleted" type="VoidFunction" required>
+  Function that gets invoked when the user completes the login process.
+</ParamField>
+
+<ParamField path="onRedirectToSignup" type="VoidFunction">
+  Function that redirects the user to Signup.
+</ParamField>
+
+<ParamField path="onRedirectToForgotPassword" type="VoidFunction">
+  Function that redirects the user to Forgot Password.
+</ParamField>
+
+<ParamField path="onRedirectToPasswordlessLogin" type="VoidFunction">
+  Function that redirects the user to Passwordless Login.
```

Comment:
We should add messaging for this or a call out that by default projects have passwordless turned on, something similar to Andrew's suggestion for the error message "Passwordless login is enabled by default for projects. The LoginPasswordless component is used to handle those cases. Make sure to either implement onRedirectToPasswordlessLogin and have it redirect to the page with a LoginPasswordless component, or to turn off Passwordless login for your project"